cfme/fixtures/cli.py

```diff
@@ -52,18 +53,28 @@ def fqdn_appliance(appliance):
     else:
         logger.error("Couldn't provision an appliance at all")
         raise SproutException('No provision available')
-    yield apps[0]
+    return apps[0]
 
     apps[0].ssh_client.close()
     sp.destroy_pool(pool_id)
 
 
+@pytest.yield_fixture(scope="function")
+def appliance_with_disk(appliance):
+    yield fqdn_appliance(appliance, preconfigured=False)
+
+
+@pytest.yield_fixture(scope="function")
+def ipa_appliance(appliance):
```

Comment:
**REQUIRED:** In looking at this now, we should really clean up on our language a little. We have
```fqdn_appliance```, which is turned into an ```ipa_appliance``` or an ```appliance_with_disk``` with the only difference being that we put ```preconfigure``` to ```False```, or ```True```. I know I requested the name change to ```appliance_with_disk```, but I feel we should rename to ```ipa_appliance_with_disk``` and ```ipa_appliance_without_disk``` or something to at least make a distinction that they are kinda the same, but have a difference. Or even ```ipa_appliance_preconfigured``` something that lets us see that they are part of the same family.

Reply:
We should just use preconfigured_appliance and unconfigured_appliance thats the real difference.